src/droplet.F

```diff
@@ -3203,7 +3217,7 @@ subroutine droplet_injection(pdata,xf,yf,dt,time_since_inject,mtime,my_reintro)
                end do
                rad_init = rad_init*1.0e-6
                !DHR TEST, to make sure that droplets that destabilize numerics don't get produced
-               rad_init = min(rad_init,250.0e-6)
+               !rad_init = min(rad_init,250.0e-6)
```

Comment:
remove this line if it is no longer needed.

Reply:
Fixed